salt/cloud/clouds/vmware.py

```diff
@@ -471,20 +473,29 @@ def _add_new_scsi_controller_helper(scsi_controller_label, properties, bus_numbe
     return scsi_spec
 
 
-def _add_new_ide_controller_helper(ide_controller_label, properties, bus_number):
+def _add_new_ide_controller_helper(ide_controller_label, controller_key, bus_number):
     '''
     Helper function for adding new IDE controllers
 
     .. versionadded:: 2016.3.0
+
```

Comment:
Since this is develop branch, keep the docs in this PR but the docs for internal functions will be changed soon to make them more consistent.